tfhe/src/core_crypto/entities/seeded_ggsw_ciphertext_list.rs

```diff
@@ -466,3 +467,77 @@ impl<Scalar: UnsignedInteger, C: ContainerMut<Element = Scalar>> ContiguousEntit
     where
         Self: 'this;
 }
+
+pub struct GgswCiphertextListConformanceParameters {
+    pub len: usize,
+    pub glwe_size: GlweSize,
+    pub polynomial_size: PolynomialSize,
+    pub decomp_base_log: DecompositionBaseLog,
+    pub decomp_level_count: DecompositionLevelCount,
+    pub ciphertext_modulus: CiphertextModulus<u64>,
+}
+
+impl TryFrom<&MultiBitBootstrapKeyConformanceParams> for GgswCiphertextListConformanceParameters {
```

Comment:
why is this one needed ?

Reply:
It's used in `impl ParameterSetConformant for SeededLweMultiBitBootstrapKey`